canonical-websites/snapcraft-flask
# Add ./run script

This uses the pre-release version 1.1.0 of the run script, and so reviewing this branch is in a way a QA test for https://github.com/canonical-webteam/generator-canonical-webteam/pull/35

Fixes #6

If reviewing this PR goes well, you've basically done half the job of reviewing https://github.com/canonical-webteam/generator-canonical-webteam/pull/35 also - so you may as well go and review that PR too ;)

QA
--

``` bash
./run help
./run  # Check it runs the server on port 8021
./run clean
./run watch
./run test
... etc.
```
